src/persistence/secondary/mongo-secondary-entity-manager.ts

```diff
@@ -0,0 +1,62 @@
+import { Entity, Model } from '@antjs/ant-js/build/ant';
+import { Collection, MongoClient } from 'mongodb';
+import { SecondaryEntityManager } from './secondary-entity-manager';
+
+export class MongoSecondaryEntityManager<TEntity extends Entity> implements SecondaryEntityManager<TEntity> {
+
+  model: Model<TEntity>;
+  private collection: Collection;
+
+  constructor(url: string, dbName: string, collectionName: string) {
```

Comment:
The constructor requires a model. The "collectionName" param is redundant, model.collectionName should be used instead. Maybe url and dbName could be extracted in an interface MongoBuildArgs.